docs/get-started/quick_start.md

```diff
@@ -1,8 +1,14 @@
 # Quick Start
 
-To compute on encrypted data, you first need to define the function you want to compute, then compile it into a Concrete `Circuit`, which you can use to perform homomorphic evaluation.
+This document covers how to compute on encrypted data homohorphically using the **Concrete** framework. We'll take you through a complete example step-by-step.
```

Comment:
> homohorphically 

typo

Reply:
> We'll

Is it ok? Shouldn't it be "Will"

Reply:
We'll = We will
It's Grammarly correct, but I changed it to "We will" for better clarity.